hpx/util/coroutine/detail/tss.hpp

```diff
@@ -53,6 +53,7 @@ namespace hpx { namespace util { namespace coroutines
                 value_(val)
             {}
 
+#if !(defined(HPX_INTEL_VERSION) && __GNUC__ == 4 && __GNUC_MINOR__ == 6)
```

Comment:
On 10/29/2015 01:31 PM, Hartmut Kaiser wrote:
> In hpx/util/coroutine/detail/tss.hpp
> <https://github.com/STEllAR-GROUP/hpx/pull/1830#discussion_r43378586>:
> 
>> @@ -53,6 +53,7 @@ namespace hpx { namespace util { namespace coroutines
>>                  value_(val)
>>              {}
>>  
>> +#if !(defined(HPX_INTEL_VERSION) && __GNUC__ == 4 && __GNUC_MINOR__ == 6)
> 
> We have a |HPX_GCC_VERSION| for this kind of test, I believe.

HPX_GCC_VERSION is undefined when compiling with the intel compiler.

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/STEllAR-GROUP/hpx/pull/1830/files#r43378586>.
> 


-- 
Thomas Heller
Friedrich-Alexander-Universität Erlangen-Nürnberg
Department Informatik - Lehrstuhl Rechnerarchitektur
Martensstr. 3
91058 Erlangen
Tel.: 09131/85-27018
Fax:  09131/85-27912
Email: thomas.heller@cs.fau.de